shop/src/main/java/ro/msg/learning/shop/entities/Customer.java

```diff
@@ -0,0 +1,21 @@
+package ro.msg.learning.shop.entities;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity @Data @EqualsAndHashCode(callSuper = true)
+@Table(name="customer")
+public class Customer extends BaseEntity implements Serializable {
+    private String first_name;
+    private String last_name;
```

Comment:
Java uses camel case notation so first_name should be firstName. (same for last_name)
To keep the connection to the database you can use @Column(name = "first_name") from javax.persistence. 
Here you can also declare if column is nullable(default true), unique(default false).